package.json

```diff
@@ -13,10 +14,13 @@
     "web-vitals": "^0.2.4"
   },
   "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
-    "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "start": "npm run handle \"$npm_config_env\" start",
+    "build": "npm run handle \"$npm_config_env\" build",
+    "test": "npm run handle \"$npm_config_env\" test",
+    "eject": "npm run handle \"$npm_config_env\" eject",
+    "handle": "run(){ if test \"$1\" = \"debug\" ; then npm run cScript $2; else npm run rScript $2; fi }; run",
+    "rScript": "run(){ echo 'running react-scripts '$1; react-scripts $1; }; run",
+    "cScript": "run(){ echo 'running craco '$1; craco $1; }; run"
   },
```

Comment:
would this ever get into a production codebase? 🤔

how could it be improved?
open to ideas